src/ngraph/ngraph_graph.h

```diff
@@ -94,7 +94,7 @@ class Node {
   bool in_ngraph_ = false;
 
   std::string operation_ = "";
-  int subgraph_ = 0;
+  int subgraph_ = -1;
```

Comment:
is this changed just for consistency, avoid 0 initialization issue ?

Reply:
I definitely designed it to count from 1 at the beginning, why do we feel the change is necessary?

Reply:
Ok, I'll revert back, and change graph id to count from 1 (instead of 0).